15-draft.json

```diff
@@ -176,43 +187,65 @@
                 "type": "string"
               },
               "phone": {
-                "description": "Example: <samp>['+1 800 555 4567','+1 800 555 4544']</samp>",
-                "type": "string"
+                "description": "Phone number, including country code with a leading plus sign",
+                "type": "string",
+                "examples": [
+                  "+1 800 555 4567",
+                  "+41 79 123 45 67"
+                ]
               },
               "email": {
                 "description": "Email address which can be base64 encoded.",
                 "type": "string"
               },
               "twitter": {
                 "description": "Twitter username with leading <samp>@</samp>.",
```

Comment:
Why the `<samp>` tag here?

Reply:
I guess for rendering as code. Semantically it should not be `<samp>`, I can fix that.